config/owasp/suppressions.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress>
-        <notes><![CDATA[file name: spring-retry-**.jar]]></notes>
+        <notes><![CDATA[
+   file name: spring-retry-1.3.3.jar
+   ]]></notes>
```

Comment:
insted of using hard coded version user delimiters
file name: spring-retry-**.jar

Reply:
Done